WordPress/UITestsFoundation/Screens/Editor/BlockEditorScreen.swift

```diff
@@ -186,25 +186,16 @@ public class BlockEditorScreen: ScreenObject {
     private func addMediaBlockFromUrl(blockType: String, UrlPath: String) {
         addBlock(blockType)
         insertFromUrlButton.tap()
-        app.textFields.element.typeText(UrlPath)
+        type(text: UrlPath, in: app.textFields.element)
         // to dismiss media block URL prompt
         tapTopOfScreen()
     }
 
-    // Can be removed once `testAddMediaBlocks()` test is fixed on iPhone
-    @discardableResult
-    public func verifyImageBlockDisplayed() -> Self {
-        let imagePredicate = NSPredicate(format: "label == 'Image Block. Row 1'")
-        XCTAssertTrue(app.buttons.containing(imagePredicate).firstMatch.waitForExistence(timeout: 5))
-
-        return self
-    }
-
     @discardableResult
     public func verifyMediaBlocksDisplayed() -> Self {
-        let imagePredicate = NSPredicate(format: "label == 'Image Block. Row 1'")
-        let videoPredicate = NSPredicate(format: "label == 'Video Block. Row 2'")
-        let audioPredicate = NSPredicate(format: "label == 'Audio Block. Row 3'")
+        let imagePredicate = NSPredicate(format: "label == 'Image caption. Empty'")
+        let videoPredicate = NSPredicate(format: "label == 'Video caption. Empty'")
+        let audioPredicate = NSPredicate(format: "label == 'Audio Player'")
```

Comment:
Any particular reason this was updated? In the previous version the Row number was added so we'd know that the blocks are added and will maintain in the order that they were added.

Reply:
Thanks for explaining the reason (and sorry I missed the commit message earlier 🙏), I checked and you're right, though I think it would be good to get the test to verify the order too. I would vote for the assertions to cover both ordering and media attachments.

Reply:
Thanks for the feedback. I addressed this https://github.com/wordpress-mobile/WordPress-iOS/pull/21639/commits/d0a0f2ab026134e46995de9b38cbc7d6a1c6c735.